File: YAML/02_load/go.py
#! /usr/bin/env ruby

require 'yaml'

puts "hello"

food = YAML.load_file('food.yaml')

if food.class == Hash 
  food.each do | key, value |
    puts "#{key} : "

    if value.class == Array
      value.each do | item |
        puts "   #{item}"
      end
    else
      puts "   #{value}"
    end
  end
else
  puts "Food is not Hash"
  puts food
end


puts "bye"

File: script_location/go.rb
#! /usr/bin/env ruby


puts "hello"

puts "__FILE__ is #{__FILE__}"
puts "File::expand_path(__FILE__) is #{File::expand_path(__FILE__)}"

puts "File::expand_path(File::dirname(__FILE__)) is #{File::expand_path(File::dirname(__FILE__))}"


